### PR TITLE
Review process

### PR DESCRIPTION
**Pull request by INRokh**
1. Added new component Review List for admin
2. Added submit functionality for user
3. Added approve, reject annotation review functionality for admin

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -65,12 +66,17 @@ class App extends Component {
       return (
         <BrowserRouter>
           <Link to="/">Images</Link> | <Link to="/users">Users</Link> |
+          <Link to="/annotations">Review List</Link> |
```
Comment (INRokh) [inline src/App.js:69]: new page for review

File: src/components/AnnotationList.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ class AnnotationList extends Component {
                   <div> Lot: {image.lot}</div>
                   <div> Unit Number: {image.unitNumber}</div>
                   <div> Product Description:{image.productDescription}</div>
+                  <div> Status:{annotation.status}</div>
```
Comment (INRokh) [inline src/components/AnnotationList.js:26]: show annotation status

File: src/components/ImageAnnotation.js [hunk 3]
```diff
@@ -10,6 +10,10 @@ class ImageAnnotation extends Component{
 
     // update state with the coordinates
     handleNewCoordinate = (coordinates) => {
+      if (this.props.isReadOnly === true) {
```
Comment (INRokh) [inline src/components/ImageAnnotation.js:13]: logic for tags, once user submit annotation tags become disabled

File: src/components/ReviewList.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
```
Comment (INRokh) [inline src/components/ReviewList.js:1]: new component for review process

File: src/components/pages/AnnotationPage.js [hunk 5]
```diff
@@ -3,32 +3,72 @@ import { connect } from "react-redux";
...
+  const { annotation, user } = props;
   if (!annotation || annotation._id !== id) {
     props.fetchAnnotationDetails(id);
     return <p>Loading...</p>;
   }
 
   let handleNewMarks = m => props.fetchAnnotationDetails(id);
+  
+  // Submit annotation by regular user
+  let handleSubmit = e => LocalApi.patch("/annotations/" + id + "/review/")
```
Comment (INRokh) [inline src/components/pages/AnnotationPage.js:19]: logic for review

**Review by EllieChen-Git — APPROVED** [review]

**Comment by EllieChen-Git** [comment]
All good. Thanks!

**Pull request merged**